show-land/sass/layout/_page-builder.scss

```diff
@@ -0,0 +1,66 @@
+.page-builder {
+  display: flex;
+  gap: 3.4rem;
+  padding-top: 5rem;
+  margin-bottom: 10rem;
+  &__icone {
+    stroke: $colorWhite;
+    background-color: $colorSeventh;
+    padding: 0.2rem 0.2rem 0.2rem 0.2rem;
+    width: 1.7rem;
+    height: 1.7rem;
+    border-radius: 10rem;
+    margin-bottom: -0.4rem;
+  }
+  &__button {
+    display: inline-flex;
+    align-items: baseline;
+    border-radius: 10rem;
+    padding: 0 1rem 0 1rem;
```

Comment:
I suggest you write like this:
padding : 0 1rem :)

Reply:
and you can use our variables. ;)
like this :
padding: list.nth($spaces, 4) 0